test_project/store/src/main/java/test/sergeyvalyushko/store/helpers/XmlParser.java

```diff
@@ -0,0 +1,34 @@
+package test.sergeyvalyushko.store.helpers;
+
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.fasterxml.jackson.dataformat.xml.XmlMapper;
+
+import java.io.*;
+
+public class XmlParser {
+    private String value;
+    public String parseFromXml(String input) throws IOException {
+        File file = new File("store/src/main/resources/config.xml");
+        String xml = inputStreamToString(new FileInputStream(file));
+        ObjectNode node = new XmlMapper().readValue(xml, ObjectNode.class);
```

Comment:
i can be wrong, but i think xmlMapper can handle File as input param, no need of additional convertion